dev/Soniyanayak51/classificationModelFunctions.py

```diff
@@ -0,0 +1,175 @@
+    df['EDUCATION'] = np.where(df['EDUCATION'] == '0', '4', df['EDUCATION'])
+    df['MARRIAGE'] = np.where(df['MARRIAGE'] == '0', '3', df['MARRIAGE'])
+
+
+def plotGrapthForDefaults(y):
+    '''
+    Plots a graph to show default values
+    Agrs: The label y
+    '''
+
+    fig = plt.figure()
+    ax = fig.add_axes([0, 0, 1, 1])
+    index, counts = np.unique(y, return_counts=True)
+    ax.bar(index, counts)
+    plt.show()
+
+
+def plotGrapthForAttributes(df):
+    '''
+    Plots Graphs for each attibute for EDA
+    Args: The dataframe
+    '''
+    # Creating a new dataframe with categorical variables
+    subset = df[['SEX', 'EDUCATION', 'MARRIAGE', 'PAY_0', 'PAY_2', 'PAY_3', 'PAY_4',
+                 'PAY_5', 'PAY_6', 'default payment next month']]
+
+    f, axes = plt.subplots(3, 3, figsize=(20, 15), facecolor='white')
+    f.suptitle('FREQUENCY OF CATEGORICAL VARIABLES (BY TARGET)')
+    ax1 = sns.countplot(x="SEX", hue="default payment next month", data=subset, palette="Blues", ax=axes[0, 0])
+    ax2 = sns.countplot(x="EDUCATION", hue="default payment next month", data=subset, palette="Blues", ax=axes[0, 1])
+    ax3 = sns.countplot(x="MARRIAGE", hue="default payment next month", data=subset, palette="Blues", ax=axes[0, 2])
+    ax4 = sns.countplot(x="PAY_0", hue="default payment next month", data=subset, palette="Blues", ax=axes[1, 0])
+    ax5 = sns.countplot(x="PAY_2", hue="default payment next month", data=subset, palette="Blues", ax=axes[1, 1])
+    ax6 = sns.countplot(x="PAY_3", hue="default payment next month", data=subset, palette="Blues", ax=axes[1, 2])
+    ax7 = sns.countplot(x="PAY_4", hue="default payment next month", data=subset, palette="Blues", ax=axes[2, 0])
+    ax8 = sns.countplot(x="PAY_5", hue="default payment next month", data=subset, palette="Blues", ax=axes[2, 1])
+    ax9 = sns.countplot(x="PAY_6", hue="default payment next month", data=subset, palette="Blues", ax=axes[2, 2]);
+
+
+def splitData(df, features, y):
+    '''
+    Splits the data into test and train
+    Args: The main dataframe, label y and the features
+    Returns: X_train, X_test, y_train, y_test the training and testing datasets respectively
+    '''
+    X_train, X_test, y_train, y_test = train_test_split(features, y, test_size=0.30, random_state=42)
+    # X_cv, X_test, y_cv, y_test = train_test_split(X_test, y_test, test_size=0.67, random_state=42)
+    # X_cv.shape
+    return X_train, X_test, y_train, y_test
+
+
+def standardize(x, x_train):
+    '''
+    Utility function for standardizeData()
+    standardizes according to train_data mean and std
+    Args: x - data to be standardize
+          x_train - trainind data
+    Returns: standardized Data x
+    '''
+    x = x.astype(float)
+    x_train = x_train.astype(float)
+    for column in x.columns:
+        mean = np.mean(x_train[column])
+        std = np.std(x_train[column])
+        x[column] = (x[column] - mean) / std
```

Comment:
I believe this standardization can be done more easily using `StandardScaler`, which I see you imported above. I'd recommend using that unless there is a reason why it didn't work for you.